src/coreclr/jit/emit.cpp

```diff
@@ -4818,10 +4821,10 @@ void emitter::emitCheckAlignFitInCurIG(unsigned short nAlignInstr)
 //-----------------------------------------------------------------------------
 //
 //  The next instruction will be a loop head entry point
-//  So insert an alignment instruction here to ensure that
-//  we can properly align the code.
+//  So insert an alignment instruction of "paddingBytes" to ensure that
+//  the code is properly aligned.
```

Comment:
Explain the `isFirstAlign` argument here (preferably with the new-style function header comment)